### PR TITLE
🚧 Fix 1password crash on iPad

### LINKED ISSUE #4942: iPad: Password manager login is broken
The app crashes when attempting to use the password manager login.

### PR DESCRIPTION
**Pull request by sofokli1**
## What's this?
Fixes crash when tapping on 1password button on ipad

### Relationships
Closes #4942 

## Why do we want this?
No 💥

## How is it done?
I added an extra field to the `OnePasswordServiceIos` to store the source view

### Why not another way?
Other way is to just hide the button on pad

### Side effects
none

## Review hints
run on device if you have one or just approve blindly.

## :squid: Permissions
pad crew

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs [hunk 1]
```diff
@@ -35,12 +35,12 @@ public enum ShakeTargets
         private readonly IAnalyticsService analyticsService;
         private readonly IOnboardingStorage onboardingStorage;
         private readonly IMvxNavigationService navigationService;
-        private readonly IPasswordManagerService passwordManagerService;
         private readonly IErrorHandlingService errorHandlingService;
         private readonly ILastTimeUsageStorage lastTimeUsageStorage;
         private readonly ITimeService timeService;
         private readonly ISchedulerProvider schedulerProvider;
         private readonly IRxActionFactory rxActionFactory;
+        public readonly IPasswordManagerService PasswordManagerService;
```
Comment (simonrozsival) [inline Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs]: We don't do public fields, we do public properties.
```suggestion
        public IPasswordManagerService PasswordManagerService { get; }
```

File: Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs [hunk 2]
```diff
@@ -17,6 +16,7 @@
 using Toggl.Multivac.Extensions;
 using Toggl.PrimeRadiant.Settings;
 using Toggl.Ultrawave.Exceptions;
+using System.Linq;
```
Comment (simonrozsival) [inline Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs]: What do we need `Linq` for? 🤔

Comment (sofokli1) [reply Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs]: will remove

**Comment by dshr** [comment]
CI builds fail on this, can you merge back from the main project branch?

**Comment by sofokli1** [comment]
@dshr I can't approve this since I created this. But I approve of the changes. At this point we just need @heytherewill to approve and is good for squashing.

**Pull request merged**